rest-api-spec/src/main/resources/rest-api-spec/api/simulate.ingest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "simulate.ingest":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/simulate-ingest-api.html",
+      "description":"Simulates running ingest with example documents."
+    },
+    "stability":"experimental",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_ingest/_simulate",
+          "methods":[
+            "GET",
+            "POST"
+          ]
+        }
```

Comment:
I think this needs to add the extra path (with `{index}`) now that you added it

Reply:
Done